src/lightning_app/components/serve/auto_scaler.py

```diff
@@ -463,7 +467,9 @@ def send_request_to_update_servers(self, servers: List[str]):
             "username": USERNAME,
             "Authorization": AUTHORIZATION_TYPE + " " + data,
         }
-        response = requests.put(f"{self.url}/system/update-servers", json=servers, headers=headers, timeout=10)
+        response = requests.put(
```

Comment:
if the `internal_ip` is none  just return from here without making a get request that would eventually crash